### PR TITLE
Add CopyAcrImages command to support fast coping within Azure

### PR DESCRIPTION
**Pull request by MichaelSimons**
This is being added to support moving to our Docker repos to MCR.

### CODE REVIEW

File: Microsoft.DotNet.ImageBuilder/src/Commands/CopyAcrImagesCommand.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                    .SelectMany(image => image.ActivePlatforms)
+                    .SelectMany(platform => platform.Tags);
+                string fullRegistryName = $"{Options.Registry}.azurecr.io/";
+
+                foreach (TagInfo platformTag in platformTags)
+                {
+                    string sourceImage = $"{Options.SourceRepository}:{platformTag.Name}";
+                    string destImage = platformTag.FullyQualifiedName.TrimStart(fullRegistryName);
+                    helper.ExecuteAzCommand(
+                        $"acr import -n {Options.Registry} --source {sourceImage} -t {destImage} --force",
```
Comment (dseefeld) [inline Microsoft.DotNet.ImageBuilder/src/Commands/CopyAcrImagesCommand.cs]: Are you always going to want this to be --force? Or should that be an option?

File: Microsoft.DotNet.ImageBuilder/src/StringExtensions.cs [hunk 2]
```diff
@@ -6,9 +6,24 @@ namespace Microsoft.DotNet.ImageBuilder
 {
     public static class StringExtensions
     {
-        public static string TrimEnd(this string source, string value)
+        public static string TrimEnd(this string source, string trimString)
         {
-            return source.EndsWith(value) ? source.Remove(source.LastIndexOf(value)) : source;
+            while (source.EndsWith(trimString))
```
Comment (dseefeld) [inline Microsoft.DotNet.ImageBuilder/src/StringExtensions.cs]: I don't understand the usage of "while" here.  If there are multiple occurrences of trimString, they will all be removed?  When do you need that?

Comment (MichaelSimons) [reply Microsoft.DotNet.ImageBuilder/src/StringExtensions.cs]: I thought it was beneficial to mirror the existing `TrimStart(char)` and `TrimEnd(char)` methods from the Fx which trim multiple occurrences.  I thought this was valuable given they are named the same and are extension methods.  The current usage does't require this functionality.  I could be convinced to modify it if you feel strongly.

**Comment by MichaelSimons** [comment]
@dseefeld - Thanks for reviewing!

**Pull request merged**